src/Symfony/Component/PropertyAccess/Tests/StringUtilTest.php

```diff
@@ -21,7 +21,9 @@ public function singularifyProvider()
         // see http://www.scribd.com/doc/3271143/List-of-100-Irregular-Plural-Nouns-in-English
         return array(
             array('accesses', 'access'),
+            array('access', 'access'),
```

Comment:
I don't understand. `access` is not a plural but a singular. Here, we area testing the conversion of a plural to a singular, so it looks wrong to me. The same actually goes for all your other test cases.